pkg/flow/memory.go

```diff
@@ -27,6 +28,10 @@ type instanceMemory struct {
 
 	// stores the events to be fired on schedule
 	eventQueue []string
+
+	invTags map[string]string
+	o       string
```

Comment:
These field names are way too short to understand what they mean outside of the context of this PR.